include/deal.II/lac/read_write_vector.templates.h

```diff
@@ -250,7 +250,46 @@ namespace LinearAlgebra
     return *this;
   }
 
+  namespace internal
+  {
+    // In the import_from_ghosted_array_finish we need to calculate the maximal
+    // and minimal value for the given number type, which is not straight
+    // forward for complex numbers. Therefore, comparison of complex numbers is
+    // prohibited and throw an assert.
+    template <typename Number>
+    Number
+    get_min(const Number a, const Number b)
+    {
+      return std::min(a, b);
+    }
 
+    template <typename Number>
+    std::complex<Number>
+    get_min(const std::complex<Number> a, const std::complex<Number>)
+    {
+      AssertThrow(false,
+                  ExcMessage("VectorOperation::min max not"
+                             "implemented for complex numbers"));
+      return a;
+    }
+
+    template <typename Number>
+    Number
+    get_max(const Number a, const Number b)
+    {
+      return std::max(a, b);
+    }
+
+    template <typename Number>
+    std::complex<Number>
+    get_max(const std::complex<Number> a, const std::complex<Number>)
+    {
+      AssertThrow(false,
+                  ExcMessage("VectorOperation::min max not "
+                             "implemented for complex numbers"));
```

Comment:
The same issue as for the other place you just fixed. :wink: 
Please only reference one of the two operations.